From be22a31f97f265116531228bebe37e78b3be7672 Mon Sep 17 00:00:00 2001
From: Sugar Zhang <sugar.zhang@rock-chips.com>
Date: Mon, 20 Jun 2016 14:47:44 +0800
Subject: [PATCH] ARM64: dts: rk3399: add another pinctrl for spdif

rk3399 has two spdif pinctrl, product need select the one
based on the design.

Change-Id: I5c1d61dd591658b8ffd445dfb27938c8a081c058
Signed-off-by: Sugar Zhang <sugar.zhang@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3399.dtsi | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399.dtsi b/arch/arm64/boot/dts/rockchip/rk3399.dtsi
index ea1a7a1491f5..d1943241334e 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399.dtsi
@@ -1792,6 +1792,11 @@
 				rockchip,pins =
 					<4 21 RK_FUNC_1 &pcfg_pull_none>;
 			};
+
+			spdif_bus_1: spdif-bus-1 {
+				rockchip,pins =
+					<3 16 RK_FUNC_3 &pcfg_pull_none>;
+			};
 		};
 
 		spi0 {
-- 
2.35.3

